File: C#PartTwo/Homework/Methods/EnglishDigit/EnglishDigit.cs
namespace EnglishDigit
{
    using System;

    class EnglishDigit
    {
        static void Main(string[] args)
        {
            int number = int.Parse(Console.ReadLine());
            string lastDigit = GetLastDigitInEnglish(number);

            Console.WriteLine(lastDigit);
        }

        private static string GetLastDigitInEnglish(int number)
        {
            string[] digitInEnglish = { "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine" };
            int lastDigit = number % 10;
            string lastDigitAsEnglish = digitInEnglish[lastDigit];

            return lastDigitAsEnglish;
        }
    }
}

File: JavaScript-Fundamentals/Homeworks/08. Functions/03. English digit/EnglishDigit.js
function solve(args) {
    var num = +args[0],
        digit;

    digit = num % 10;

    switch (digit) {
        case 1:
            return 'one';
            break;
        case 2:
            return 'two';
            break;
        case 3:
            return 'three';
            break;
        case 4:
            return 'four';
            break;
        case 5:
            return 'five';
            break;
        case 6:
            return 'six';
            break;
        case 7:
            return 'seven';
            break;
        case 8:
            return 'eight';
            break;
        case 9:
            return 'nine';
            break;
        case 0:
            return 'zero';
            break;
    }
}